js/src/events.ts

```diff
@@ -187,6 +197,14 @@ class EventModel extends WidgetModel {
                     break
             }
         }
+        // Also add listeners to support populating the x/y traits
+        if (this.get('xy_trait_coords')) {
+            let prevent_default = this.get('prevent_default_action')
+            let handler = this._set_xy.bind(this, view)
+            let event = 'mousemove'
```

Comment:
Unless I'm missing something, these should be `const`, not `let`.

Reply:
Is it just that the last one (`event`) that should be `const`, or all three of them?